### PR TITLE
HDDS-4316. Upgrade to angular 1.8.0 due to CVE-2020-7676

### PR DESCRIPTION
**Pull request by vivekratnavel**
## What changes were proposed in this pull request?

Upgrade angular from 1.7.9 -> 1.8.0

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4316

## How was this patch tested?

Tested with docker-compose and verified that all UIs work fine as expected after the upgrade.

### CODE REVIEW

File: hadoop-ozone/pom.xml [hunk 1]
```diff
@@ -294,7 +294,7 @@
             <exclude>src/test/resources/ssl/*</exclude>
             <exclude>src/main/compose/ozonesecure/docker-image/runner/build/apache-rat-0.12/README-CLI.txt</exclude>
             <exclude>src/main/compose/ozonesecure/docker-image/runner/build/apache-rat-0.12/README-ANT.txt</exclude>
-            <exclude>webapps/static/angular-1.7.9.min.js</exclude>
+            <exclude>webapps/static/angular-1.8.0.min.js</exclude>
             <exclude>webapps/static/angular-nvd3-1.0.9.min.js</exclude>
             <exclude>webapps/static/angular-route-1.7.9.min.js</exclude>
```
Comment (dineshchitlangia) [inline hadoop-ozone/pom.xml:299]:
```suggestion
            <exclude>webapps/static/angular-route-1.8.0.min.js</exclude>
```

**Review by dineshchitlangia — CHANGES_REQUESTED** [review]
@vivekratnavel Overall LGTM, missed a pom.xml change. Suggested change inline.

**Comment by vivekratnavel** [comment]
@dineshchitlangia Good catch! Thanks for the review!

**Review by adoroszlai — APPROVED** [review]
Thanks @vivekratnavel for taking care of this.  Checked SCM, OM web, they looked OK.

**Pull request merged**